examples/solid-client-access-grants-demo/README.md

```diff
@@ -0,0 +1,49 @@
+# Solid Access Requests and Access Grants Demo
+
+## Run the demo app
+
+- Run `npm ci` and `npm run build` in the current directory to build the demo app.
+- Create a `.env.local` file based on `.env.local.example`.
+- Fill in the required settings following the instructions in `.env.local.example`.
+- Start the app `npm start`.
+
+## Run the demo app using the local library
+
+- Instead of `npm run build`, use `npm run build:local` to build the demo app using
+  the local `solid-client-access-grant-js` code (instead of the installed dependency
+  `@inrupt/solid-client-access-grants`).
+- All other steps remain the same as in "[Run the app](#run-the-app)"
+
+## Access flow
+
+Let the application be ACME.
+Let the user be Alice.
+Let the access management app be PodBrowser).
+Let X be a resource Alice owns (it is a resource in Alice's Pod).
+
+1. ACME has a WebID and a set of client credentials.
+2. Alice also has a WebID and uses PodBrowser for access management.
+3. Alice wants to share access to X with ACME.
+4. Alice fills ACME's create Access Request form.
+5. The application needs to know:
+   - Alice's WebID in order for Alice to have access to ACME's Access Request, and,
```

Comment:
remove the ", and," at the end of this line

Reply:
Addressed in https://github.com/inrupt/solid-client-access-grants-js/pull/252/commits/393f0b4ee8cc698a7eac3f2e5cd850b28d784840